runtime/src/assets_config.rs

```diff
@@ -38,7 +38,9 @@ parameter_types! {
 
 impl pallet_nfts::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
+	// TODO: source from primitives
```

Comment:
Do we want to address the TODOs before merging?

Reply:
Need to refactor primitives as noted in the PR description and didnt want it to affect any work that Ale might be busy with right now.

Reply:
I'm not using primitives for anything cross-chain so far

Reply:
It should be a quick and small PR, so better to address separately imo.